src/components/ProfileDropdown.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import {
+  Avatar,
+  Container,
+  Divider,
+  Grid,
+  Typography,
+} from '@material-ui/core';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import NewIcon from '@material-ui/icons/NewReleases';
+import BookmarkIcon from '@material-ui/icons/Bookmark';
+import DoneIcon from '@material-ui/icons/Done';
+import ExitToAppIcon from '@material-ui/icons/ExitToApp';
+
+// hacky way to override Menu style
+const StyledMenu = withStyles({
+  paper: {
+    border: '1px solid black',
+    width: '300px',
+    background: '#f2f5fa',
+  },
+})(Menu);
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    marginTop: '10px',
+    marginBottom: '15px',
+  },
+  menuList: {
+    marginTop: '10px',
+    marginBottom: '10px',
+  },
```

Comment:
Use `theme.spacing(1)`, `theme.spacing(2)`, etc. for spacing properties like `marginTop`

Reply:
Ok